model-optimizer/mo/ops/strided_slice.py

```diff
@@ -126,21 +126,27 @@ def infer(node: Node):
         node.ellipsis_mask[idx] = 0
 
         if node.graph.graph['layout'] == 'NHWC' and node.out_port(0).data.get_value() is None:
-            PermuteAttrs.create_permute_attrs(node, attrs=[('shrink_axis_mask', 'input:0', permute_masks),
-                                                           ('new_axis_mask', 'input:0', permute_masks),
-                                                           ('ellipsis_mask', 'input:0', permute_masks),
-                                                           ('begin_mask', 'input:0', permute_masks),
-                                                           ('end_mask', 'input:0', permute_masks),
-                                                           ])
-            # permute inputs
             in_shape = node.in_port(0).get_source().data.get_shape()
             assert in_shape is not None, \
                 'Input shape is unknown for 0 input of node {}'.format(node.name)
             input_rank = len(in_shape)
-            if input_rank > 3:
+            output_rank = len(out_shape)
```

Comment:
The change in this transformation is very risky. We need to run the validation with a much bigger set of models.

Reply:
I think we start such validation. All comments are addressed.